chat/fe/src/views/RoomView.ts

```diff
@@ -46,17 +58,57 @@ export function RoomView(props: RoomViewProps) {
     const newMessage = Message(message);
     messageList.prepend(newMessage);
   });
+  onDeleteMessage(props.roomId, async (messageId) => {
+    const messageToDelete = byId(messageId);
+    messageToDelete.remove();
+    const isLastMessageInList =
+      messageList.firstElementChild.classList.contains('date-divider');
+
+    if (isLastMessageInList) {
```

Comment:
What is this doing?  Seems like it's refetching room messages if there are no more messages?

Reply:
@adamjberg This is what I was trying to avoid here:
- First delete action: Not showing the date divider if you delete the first message of today's date and then sending a message again after deletion
- Second delete action: At the end of the video it shows like the "Adioooos" message disappears at the same time the "hello" is deleted because message list hasn't been updated so "Adioooos" is the last one

Didn't want to refetch so if there's a better way to handle this lmk! 

https://user-images.githubusercontent.com/66710501/167195404-816a448b-1bc1-494b-a07f-5bf46f4886d2.mov